### PR TITLE
Enable refresh button on homepage

### LINKED ISSUE #2242: Drives widget storage details don't update without reopen
**Describe the bug**
Drives widget storage details don't update without reopen of Files.

**To Reproduce**
1. Go to the Home page.
2. Note the size of a drive.
image
3. Change the files in that drive (delete everything, add a big file, etc.)
image
4. Go back to the Home page. The size won't be updated no matter if you switch directories. Only updates if you restart/open a new window.
image

**Expected behavior**
Drives should refresh on navigation to the Home page.

**Desktop:**
 - OS Version: Windows 10 2004 20246.1
 - App version: v0.19.0.0

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #2242 

**Details of Changes**
Add details of changes here.
- Added ability to refresh the widgets using the Refresh toolbar button on homepage.
- Updates drives occupied space and recent items.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.Uwp/UserControls/AddressToolbar.xaml [hunk 1]
```diff
@@ -411,7 +411,7 @@
                     Height="32"
                     x:FieldModifier="public"
                     AccessKey="R"
-                    AutomationProperties.Name="{helpers:ResourceString Name=NavRefreshButton/AutomationProperties/Name}"
+                    AutomationProperties.Name="{helpers:ResourceString Name=NavResfreshButton/AutomationProperties/Name}"
```
Comment (gave92) [inline src/Files.Uwp/UserControls/AddressToolbar.xaml:414]: This resource string is defined as NavRe*s*freshButton. A separate PR should change all tranlations files to NavRefreshButton.

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**